Jenkinsfile_k8s

```diff
@@ -34,6 +34,28 @@ pipeline {
             }
           }
         }
+        stage('GC on Azure') {
+          environment {
+            PKR_VAR_azure_subscription_id = credentials('packer-azure-subscription-id')
+            PKR_VAR_azure_client_id       = credentials('packer-azure-client-id')
+            PKR_VAR_azure_client_secret   = credentials('packer-azure-client-secret')
```

Comment:
If you have the time, take a look at https://github.com/jenkins-infra/packer-images/issues/183 to use an azure credentials instead.

(Could be done in another PR)

Reply:
will be done in another PR, but you are correct :)